dev/create-release/release-build.sh

```diff
@@ -220,7 +220,7 @@ git clean -d -f -x
 rm -f .gitignore
 cd ..
 
-export MAVEN_OPTS="-Xss128m -Xmx12g"
+export MAVEN_OPTS="-Xss128m -Xmx12g -XX:ReservedCodeCacheSize=1g"
```

Comment:
`ReservedCodeCacheSize` is consistent with https://spark.apache.org/docs/latest/building-spark.html#setting-up-mavens-memory-usage.